NeoSwift/Models/TransactionAttritbute.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  TransactionAttritbute.swift
+//  NeoSwift
+//
+//  Created by Apisit Toompakdee on 9/12/17.
+//  Copyright © 2017 drei. All rights reserved.
+//
+
+import UIKit
+
+public class TransactionAttritbute: NSObject {
+    
+    enum AttributeError: Error {
+        case TooLongDescription
+    }
+    
+    enum Usage : UInt8 {
+        case contractHash = 0x00
```

Comment:
If you know the usecase of each usage can you write a small comment on the side? If not, that's ok we'll figure out later